ingestion_tools/scripts/data_validation/tests/test_frame.py

```diff
@@ -52,17 +52,54 @@ def test_extensions(self, frames_files: List[str]):
     def test_consistent(self, frames_headers: Dict[str, Union[List[tifffile.TiffPage], MrcInterpreter]]):
         return helper_tiff_mrc_consistent(frames_headers)
 
-    ### END Self-consistency tests ###
+    ### BEGIN MDOC tests ###
+    @allure.title("Mdoc: tilt angles are within the expected range [-90, 90].")
+    def test_frames_mdoc_range(self, frames_mdoc: pd.DataFrame):
+        assert frames_mdoc["TiltAngle"].min() >= -90
+        assert frames_mdoc["TiltAngle"].max() <= 90
+
+    @allure.title("Mdoc: number of mdoc tilt angles equals number of frames files.")
+    def test_mdoc_frames(self, frames_mdoc: pd.DataFrame, frames_files: List[str]):
+        assert len(frames_mdoc) == len(frames_files)
+
+    @allure.title("Mdoc: Every mdoc tilt angle matches a frames file (one-to-one).")
+    def test_mdoc_frame_paths(
```

Comment:
Should this be a blocking test or should this be categorized as a warning?

Reply:
This should be a blocking validation. We should have a frame file for every mdoc section.